app/partials/app.cjsx

```diff
@@ -72,7 +70,7 @@ module.exports = React.createClass
   getEnv: ->
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env || 'staging'
+    @state?.env || browser_env[1] || 'staging'
```

Comment:
Oh no, I wrote it. I'm just not sure if this is the best method for getting the environment.